src/lightbox-manager.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {getExistingServiceForDoc} from './service';
+
+/*
+ * @return {!../extensions/amp-lightbox-viewer/service/lightbox-manager-impl.LightboxManager}
+ **/
+export function lightboxManagerForDoc(nodeOrDoc) {
```

Comment:
Let's remove this module for now. It won't work correctly unfortunately since we currently don't have a way to declare an element-service dependency for doc services. Since we are only using it inside of the `amp-lightbox-viewer` package, we can simply use `install` method there. We'll have to figure it out.